internal/provider/resource_schema_latest_refresh_test.go

```diff
@@ -267,6 +267,28 @@ func testAccCheckLatestRefreshSchemaConfig(confluentCloudBaseUrl, mockServerUrl
         subject_name = "%s"
         version = %d
       }
+  	  ruleset {
```

Comment:
If it's not present already, is it possible to update one of the acceptance rulesets to include a test case for the "update" operation, which involves removing a ruleset?